h/models/job.py

```diff
@@ -85,6 +85,5 @@ def __repr__(self):
                 "expires_at",
                 "priority",
                 "tag",
-                "kwargs",
```

Comment:
Removed `Job.kwargs` from `Job.__repr__()` so it doesn't get logged: I think `Job.kwargs` could sometimes contain sensitive information. In general I think the `__repr__()`'s of model objects should be quite conservative: include the primary key and only include other attributes if they're definitely always going to be short and never contain any sensitive information.

Reply:
Yeah I agree, I reckon it should be enough to uniquely identify a row/action, not to recover all the data from the logs.